Zype/View Controllers/PlayerVC.swift

```diff
@@ -215,6 +226,21 @@ class PlayerVC: UIViewController, DVIABPlayerDelegate {
                 }
             }
         }
+
+        let analyticsEmpty = self.analyticsInfo.isEmpty
+
+        if !analyticsEmpty {
+            var analyticsData = self.analyticsInfo
+            analyticsData["beacon"] = nil
```

Comment:
This seems hacky, setting one of the parameters to nil. It's not clear what is happening.